client/payment-gateways/__tests__/index.test.js

```diff
@@ -0,0 +1,38 @@
+/**
+ * External dependencies
+ */
```

Comment:
We don't need the `External dependencies`/`Internal dependencies` in the imports, they can all be removed from these files
```suggestion
```

Reply:
Noted. Removed them everywhere I had them.